config.txt

```diff
@@ -47,3 +47,5 @@ enable_uart=1
 #dtoverlay=vc4-fkms-v3d
 #max_framebuffers=2
 
+# Enable CAN Bus
+dtoverlay=mcp2515-can0,oscillator=8000000,interrupt=25
```

Comment:
CAN would be USB-attached, since on PLT-300A the RPi4 will only interface through USB and HDMI with the rest of the PLT.